docker-compose.yml

```
@@ -31,3 +31,14 @@ services:
             - ./docker/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
             - ./public:/usr/src/app/public
         restart: unless-stopped
+    db:
+        image: mysql
+        command: --default-authentication-plugin=mysql_native_password
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: No!ss@r2022
+    adminer:
+        image: adminer
+        restart: always
```

La _restart policy always_ ferait redémarrer ta base de données tout le temps au redémarrage de ton PC. Ça peut être embêttant

```suggestion
        restart: unless-stopped
```